modules/nw-egress-ips-namespaces.adoc

```diff
@@ -0,0 +1,68 @@
+// Module filename: nw-egress-ips-namespaces.adoc
+//
+// Module included in the following assemblies:
+// * networking/managing-networking.adoc
+
+[id='nw-egress-ips-namespaces-{context}']
+= Enabling Automatic Egress IPs
+
+As a cluster administrator, you can assign
+egress IP addresses to namespaces by setting the `egressIPs` field to the
+`NetNamespace` resource.
+
+With fully automatic egress IPs, you can set the `egressCIDRs` field
```

Comment:
s/egress IPs/egress IP address assignment? You need a noun here. And in the title.